### PR TITLE
DEV: Remove unused fixture code

### PR DESCRIPTION
**Pull request by CvX**

### CODE REVIEW

File: test/javascripts/chat-fixtures.js [hunk 1]
```diff
@@ -196,7 +195,7 @@ export const chatChannels = {
 function addSettingsAttrs(channel) {
   channel.current_user_membership = channel.current_user_membership || {};
   channel.current_user_membership.unread_count = 0;
-  channel.current_user_membership.unrad_mentions = 0;
+  channel.current_user_membership.unread_mentions = 0;
```
Comment (tgxworld) [inline test/javascripts/chat-fixtures.js:198]: Does this typo not affect any features?

Comment (jjaffeux) [reply test/javascripts/chat-fixtures.js:198]: Most of unread stuff is very badly tested, so iI wouldn't be surprised, this is just setting some general stuff and probably most of it is not used. We can keep it though as it's a valid expectation to have this here, will remove it once we move to fabricators.

**Review by jjaffeux — APPROVED** [review]

File: test/javascripts/chat-fixtures.js (hunk 1 above)
Comment (CvX) [reply test/javascripts/chat-fixtures.js]: Hm, actually the whole thing seems to be unused 👀

**Comment by jjaffeux** [comment]
I already have this removed in upcoming fabricator pr, fine for this one but please dont spend too much time hunting for this

**Review by jjaffeux — APPROVED** [review]

**Pull request merged**